src/components/ContextBar.js

```diff
@@ -145,15 +145,20 @@ export const ContextBar = ({
     }),
     div({ style: { ...Style.elements.contextBarContainer, width: 70 } }, [
       div({ style: contextBarStyles.contextBarContainer }, [
+
         h(Fragment, [
           h(TooltipTrigger,
             {
               side: 'left',
               delay: 100,
               content: [
-                div({ key: 'p1' }, 'This rate reflects the estimated aggregate hourly cost for running and paused applications, as well as associated persistent disks. For more details, click on the Cloud icon.'),
+                div({ key: 'p1' },
+                  ['Estimated hourly rate for all applications in a running or paused state, and associated persistent disks. For details, click',
+                    img({ src: cloudIcon, style: { height: 20, paddingLeft: 5, paddingRight: 5, verticalAlign: 'text-bottom' }, alt: 'Environment Configuration' }),
```

Comment:
Concision is beneficial here, I think:
```suggestion
                    img({ src: cloudIcon, style: { height: 20, padding: '0 5px', verticalAlign: 'text-bottom' }, alt: 'Environment Configuration' }),
```
While you're here, I think "Environment Configuration Icon" would be better alt text, since this isn't the actual control but an example of it.